### PR TITLE
add q-light zerodim, zerodim 2 pol and puck

### PR DESCRIPTION
**Pull request by geirra**
Add support for Q-Light Zerodim / Zwave Puck and the unreleased Zerodim 2 pol (identical to Zerodim)

### CODE REVIEW

File: config/q-light/q-light_puck.xml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+0 - Function disabled. 
+1-32767 - time to turn off measured in seconds (1s-9.1h)
+    </Help>
+    </Value>
+  </CommandClass>
+
+  <CommandClass id="133">
+    <Associations num_groups="5">
+      <Group index="1" label="Lifeline" max_associations="1" />
+      <Group index="2" label="On/Off(S1) / Mirror of endpoint 1, group 2" max_associations="5" auto="true" />
```
Comment (Fishwaldo) [inline config/q-light/q-light_puck.xml:143]: Are you sure you need each Association Group as Auto? Does the device not send everything to lifeline? (it should)

File: config/q-light/q-light_zerodim.xml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+1-32767 - time to turn off measured in seconds (1s-9.1h)
+    </Help>
+    </Value>
+
+  </CommandClass>
+
+  <CommandClass id="133">
+    <Associations num_groups="3">
+      <Group index="1" label="Lifeline" max_associations="1" />
+      <Group index="2" label="On/Off (rotary nob) / Mirror of endpoint 1, group 2" max_associations="5" auto="true" />
```
Comment (Fishwaldo) [inline config/q-light/q-light_zerodim.xml:114]: same question as above.

File: config/q-light/q-light_zerodim_2pol.xml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+0 - Function disabled. 
+1-32767 - time to turn off measured in seconds (1s-9.1h)
+    </Help>
+    </Value>
+
+  </CommandClass>
+
+  <CommandClass id="133">
+    <Associations num_groups="3">
+      <Group index="1" label="Lifeline" max_associations="1" />
```
Comment (Fishwaldo) [inline config/q-light/q-light_zerodim_2pol.xml:113]: same question as above.

File: config/q-light/q-light_zerodim.xml (hunk 2 above)
Comment (geirra) [reply config/q-light/q-light_zerodim.xml]: yes, your correct. fixed the mistake :)

File: config/q-light/q-light_zerodim_2pol.xml (hunk 3 above)
Comment (geirra) [reply config/q-light/q-light_zerodim_2pol.xml:114]: yes, your correct. fixed the mistake :)

**Comment by geirra** [comment]
Errors corrected and branch synced with upstream, should be ready for merge now @Fishwaldo

**Pull request merged**